include/seqan/graph_align/graph_impl_align.h

```diff
@@ -1656,7 +1656,7 @@ convertAlignment(Graph<Alignment<TStringSet, TCargo, TSpec> > const& g,
     TSize col = 0;
     typename TPosToVertexMap::const_iterator it = g.data_pvMap.begin();
     unsigned int compIndex = 0;
-    unsigned int compIndexLen = length(order);
+    unsigned long compIndexLen = length(order);
     TIdType currentSeq = it->first.first;
```

Comment:
Since order is of type `String<unsigned int>` the return value of length is of type `typename Position<String<unsigned int> >::Type`.